tm2py/components/demand/prepare_demand.py

```diff
@@ -134,7 +134,7 @@ def _prepare_demand(
                  "factor": <factor to apply to demand in this file>}
             time_period (str): the time time_period ID (name)
         """
-        scenario = self.get_emme_scenario(self._emmebank_path, time_period)
+        scenario = self.get_emme_scenario(self._emmebank.path, time_period)
```

Comment:
@xyzjayne what is _emmebank.path? is this a typo?

Reply:
On our end, ``self._emmebank_path`` produces a non-existent path where the project example name gets duplicated, "Link21/Link21/emme_project/Database_highway/emmebank"
``self._emmebank.path`` returns the correct path.

Reply:
I see. Do you have the two *.toml files in the "Link21" folder or in the parent folder of "Link21"? The formal would cause this issue.

Reply:
They are in the parent folder (usually in examples).

Reply:
Reverted back - hopefully we'll find a fix in a future commit